bin/setup

```diff
@@ -59,6 +59,11 @@ if [ ! -f .env ]; then
   cp .sample.env .env
 fi
 
+# Set up Puma's process ID folder
+if [ ! -d tmp/pids ]; then
+  mkdir -p tmp/pids
```

Comment:
Puma needs `tmp/pids` to be present for this version of Rails to work.

Rather than add this to `.gitignore`, I chose to create the folder during setup. Thoughts welcome on this approach.

Ref:
- https://www.alanvardy.com/post/puma-server-not-loading